README.md

```diff
@@ -45,5 +45,7 @@ The following environment variables are available:
 ## Running tests
 
 ```bash
-docker run  -w /app -v $(pwd):/app  php:7.4-cli-alpine php /app/vendor/bin/phpunit -c /app/phpunit.xml.dist --colors=always
+$ make test
```

Comment:
It is kind of obvious fact but it will fail without running `composer install` before this.
Shouldn't we mention that you have to run "composer" before any "make" command?

Reply:
added on README, wdyt?